navigation/DrawerContent.js

```diff
@@ -24,6 +25,14 @@ class DrawerContent extends React.Component {
       } else {
         customer = { name: 'Guest' };
       }
+      Sentry.configureScope(scope => {
+        scope.setExtra('setUserScreen', 'DrawerContent');
```

Comment:
@JustinMi @wangannie Let's remove the `scope.setExtra` here as well, this should only be added when reporting a message or exception using `withScope` which will attach the `setExtra` information only within the the next call to .captureException or .captureMessage

scopes have been a journey for me to learn, here is what I'm referencing: [scopes](https://docs.sentry.io/enriching-error-data/scopes/?platform=javascript#configuring-the-scope)